### PR TITLE
DR40

### PR DESCRIPTION
**Pull request by wwysoc2**
- Release notes
- STJ known issues
- File manifests coming soon!

### CODE REVIEW

**Review by bullenc — COMMENTED** [review]
TCGA-GBM also had 1 aliquot of RNA-Seq released, HCMI-CMDC clinical supps were updated (not sure if worth mentioning), also clinical data released for TARGET projects (I probably need to add this to ORR too). 

For bugs fixed, do we need to add the FM-AD controlled mutations added back in to mutation viewer?

File: docs/Data/Release_Notes/Data_Release_Notes.md [hunk 1]
```diff
@@ -47,6 +48,78 @@
...
+* No data from TARGET-MDLS is available.
+* TCGA Projects
+    * Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265-->
+    * 74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111-->
+    * Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439-->
+    * The raw and annotated VarScan VCF files for aliquot `TCGA-VR-A8ET-01A-11D-A403-09` are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489-->
+    * Some TCGA annotations are unavailable in the Data Portal<!--DAT-52-->. These annotations can be found [here](tcga-annotations-unavailable-20170315.json).
+    * Tumor grade property is not populated <!--SV-585-->
```
Comment (clsackett) [inline docs/Data/Release_Notes/Data_Release_Notes.md:119]: Change to tumor_grade to be consistent with the bulletpoint below (or change 'Progression_or_recurrence' to 'Progression or recurrence'

**Review by clsackett — COMMENTED** [review]
One small change, otherwise LGTM

File: docs/Data/Release_Notes/Data_Release_Notes.md (hunk 1 above)
Comment (wwysoc2) [reply docs/Data/Release_Notes/Data_Release_Notes.md:120]: sgtm

**Comment by wwysoc2** [comment]
@clsackett I made the change you requested for this release.  I'm going to leave it in the previous releases since we shouldn't touch previous release notes.

@bullenc I added HCMI supplements + GBM RNA-Seq to the list.  FM-AD mutations weren't removed from the portal, they just weren't merged, so their performance was bad.  So I don't think we need to add that to the release notes.

**Review by clsackett — APPROVED** [review]
LGTM!

**Pull request merged**